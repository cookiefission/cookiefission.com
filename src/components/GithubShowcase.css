.GithubShowcase-wrapper {
  display: flex;
  width: 750px;
  margin: auto;
  margin-top: 10px;
  flex-wrap: wrap;

}

@media (max-width: 750px) {
  .GithubShowcase-wrapper {
    width: 98%;
  }
}

.GithubShowcase-repo {
  width: 48.9%;
  height: 100px;
  border: none;
  padding: 3px;
}

.GithubShowcase-content {
  height: 100%;
  border: 1px solid black;
  border-radius: 5px;
}

@media (max-width: 500px) {
  .GithubShowcase-wrapper {
    display: block;
  }

  .GithubShowcase-repo {
    width: 100%;
  }
}
