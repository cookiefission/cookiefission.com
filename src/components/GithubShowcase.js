import React from 'react';

import './GithubShowcase.css';

const GithubShowcase = (props) => {
  return(
    <div className="GithubShowcase-wrapper">
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
      <div className="GithubShowcase-repo">
        <div className="GithubShowcase-content">Foo</div>
      </div>
    </div>
  );
};

export default GithubShowcase;
