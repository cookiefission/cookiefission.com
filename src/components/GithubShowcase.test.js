import React from 'react';
import GithubShowcase from './GithubShowcase';
import renderer from 'react-test-renderer';

describe('<GithubShowcase />', () => {
  it('matches the snapshot', () => {
    const component = renderer.create(<GithubShowcase />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
