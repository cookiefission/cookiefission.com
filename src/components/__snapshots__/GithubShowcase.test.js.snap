exports[`<GithubShowcase /> matches the snapshot 1`] = `
<div
  className="GithubShowcase-wrapper">
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
  <div
    className="GithubShowcase-repo">
    <div
      className="GithubShowcase-content">
      Foo
    </div>
  </div>
</div>
`;
